app/models/okta_redis/grants.rb

```diff
@@ -8,34 +8,51 @@ module OktaRedis
   class Grants < Model
     CLASS_NAME = 'GrantsService'
 
+    after_initialize do |grants|
+      grants.instance_variable_set(:@activity_log, [])
```

Comment:
the `Model` class uses `Common::RedisStore` which defines the `after_initialize` hook.  This is mostly to tap into the "constructor" even though, which is in `RedisStore` which has a complex signature and I took this approach to be less invasive.